CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Aded
```

Comment:
```suggestion
### Added
```

Reply:
Can you just fix this tiny typo, please? I would do it myself, but I don't that have the permissions to commit on your branch

Reply:
yes, sure. My mistake :D.